pandas/tests/internals/test_internals.py

```diff
@@ -285,6 +285,14 @@ def test_delete(self):
         with pytest.raises(Exception):
             newb.delete(3)
 
+    def test_make_block_same_class(self):
+        block = create_block('M8[ns, US/Eastern]', [3])
+        with pytest.warns(FutureWarning):
+            copy = block.make_block_same_class(block.values,
+                                               dtype=block.values.dtype)
+        assert block.dtype == copy.dtype
```

Comment:
you don't need these asserts, just running this is enough

Reply:
done